packages/opentelemetry-api/src/api/propagation.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Context } from '@opentelemetry/scope-base';
+import { Carrier } from '../context/propagation/carrier';
+import { HttpTextFormat } from '../context/propagation/HttpTextFormat';
+import { NOOP_HTTP_TEXT_FORMAT } from '../context/propagation/NoopHttpTextFormat';
+
+/**
+ * Singleton object which represents the entry point to the OpenTelemetry Propagation API
+ */
+export class PropagationAPI {
```

Comment:
Should we have a common interface between `PropagationAPI` et `ContextAPI` ?

Reply:
Explain what you mean by "common interface"

Reply:
@OlivierAlbertini I just merged the context API PR and updated this to reflect it. Let me know if that was what you had in mind. #797